tasks/main.yml

```diff
@@ -1,34 +1,14 @@
 ---
-- name: Install sudo command
-  apt:
-    pkg: sudo
-    state: present
-
-- name: Create group for system administration
-  group:
-    name: "{{ sys_admin_group }}"
-    state: present
-
-- name: Create users for system administration
-  user:
-    name: "{{ item.name }}"
-    state: "{{ item.state }}"
-    shell: "/bin/bash"
-    groups: "{{ sys_admin_group }}"
-    append: yes
-  with_items: "{{ sys_admins }}"
-
-- name: Add SSH public keys to system administrators
-  authorized_key:
-    user: "{{ item.name }}"
-    key: "{{ lookup('file', item.ssh_key) }}"
-    state: "{{ item.state }}"
-  when: item.state == "present"
-  with_items: "{{ sys_admins }}"
-
-- name: Copy sudoers configuration for system administrators
-  template:
-    src: sudoers.j2
-    dest: "/etc/sudoers.d/90-sys-admins"
-    mode: 0440
-    group: "{{ sys_admin_group }}"
+- import_tasks: multiuser.yml
+  when: sysadmin_multi_user
```

Comment:
Can't we just check the length of the list of users? If >1 the import the multi.